src/style.css

```diff
@@ -0,0 +1,88 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
```

Comment:
I think these also do not do anything. your content inside of the container takes up the entire space. nothing to center

Reply:
Noted thanx. fixed